src/main/java/seedu/duke/logic/commands/module/ListModuleCommand.java

```diff
@@ -21,12 +21,16 @@ public ListModuleCommand(String argument) {
     @Override
     public void execute(Ui ui, Storage storage, TaskList taskList, LessonList lessonList, ModuleList moduleList)
             throws DukeException, IOException {
-        assert (argument.equalsIgnoreCase("verbose") | argument.isEmpty())
-                : "list module command currently only accepts empty, \"verbose\"";
-        if (argument.equalsIgnoreCase("verbose")) {
-            ui.printModulesWithDetails(moduleList);
+        if (moduleList.isEmpty()) {
+            ui.printMessage("There are no modules in the list.");
         } else {
-            ui.printModuleList(moduleList);
+            assert (argument.equalsIgnoreCase("verbose") | argument.isEmpty())
+                    : "list module command currently only accepts empty, \"verbose\"";
+            if (argument.equalsIgnoreCase("verbose")) {
+                ui.printModulesWithDetails(moduleList);
+            } else {
+                ui.printModuleList(moduleList);
+            }
```

Comment:
Hmm.. any good reason of nesting this inside an `else` rather than just using `else if`?

Reply:
This actually wasn't written by me but I see your point, I'll change this.